package.json

```diff
@@ -14,11 +14,12 @@
     "example": "examples"
   },
   "scripts": {
-    "prebuild": "rimraf dist",
+    "prebuild": "rm -rf dist/*",
     "build": "tsc",
+    "build:watch": "tsc --watch",
     "npm-publish": "npm run build && npm publish --access public",
     "npm-publish-beta": "npm run build && npm publish --access public --tag beta",
-    "npm-publish-local": "npm run build && npm publish --registry http://verdaccio:4873 --tag alpha",
+    "npm-publish-local": "npm run build && npm publish --registry http://host.docker.internal:4873 --tag alpha",
```

Comment:
Let's remember this line for a discussion how to make packages available locally.